### DISCUSSION TITLE
Web3Modal V3  Poor Theming Options

### DISCUSSION THREAD

**Discussion by C00K1E-dev** [WalletConnect/walletconnect-monorepo#4082] [Developer Support]
V3 of Web3Modal has poor theming options will there be some more options implemented ? For example the in V2 we had an option to change color of the label ...  :'--w3m-accent-fill-color': '#1a1f2c',    Even tho i tryed to define  this option in node_modules it doesnt work .... if there is an option to change the text color from button can someone let me know ?

**Comment by glitch-txs** [comment]
we'll add more theming options, any specific you'd like to have?

**Reply by C00K1E-dev** [reply]
In v2 we had the option to change the text color from the button now we don't have it 
--w3m-accent-fill-color 
Perhaps also the overlay backdrop filter and overlay background color 
Thank you for the reply tho and I eagerly wait the mods

**Reply by cpeterson42** [reply]
Would also be nice to be able to customize border radii - the rounded default works for some apps, but falls short if a UI is square/ has tighter radii

**Comment by vinaywebcom** [comment]
Same I want to change color of text of w3m-button but unable to do so

**Reply by C00K1E-dev** [reply]
You can change color of button but not color of text inside the button

**Reply by vinaywebcom** [reply]
Yes, that I did, but I want to change text color as well

**Comment by m-peko** [comment]
Is it possible to change the w3m-button height on small screen sizes? It seems the height on screen sizes less than 500px is not the same as the height of w3m-network-button

**Reply by MikhilMC** [reply]
I also have this doubt. It would have been better if we could change color.

**Comment by meetkarren** [comment]
We’d also like to change the modal background color. Not the filter, the solid black or white needs customization. Also please how do I change the button color? Talking of reown appKit modals

**Comment by motikolorado** [comment]
We need to customize the modal fully or expose hooks to create our own modal
